spec.md

```diff
@@ -0,0 +1,315 @@
+- When users open files, the host operating system typically launches the appropriate program.
+- When users open a file, from within a program, the open dialog filters to the supported types.
+- When search or security software scan the contents of a storage solution, the software must to know how to process the different types of content.
+- When users view the contents of a storage solution, they see the textual and visual indications of the type.
+
+OCI Artifacts provides these core capabilities to [OCI distribution spec][distribution-spec] based registries.
+
+Once complete, Artifacts can be identified and visualized as the following:
+
+|Icon|Artifact|`config.mediaType`|
+|-|-|-|
+|<img src="https://github.com/opencontainers/artwork/blob/master/oci/icon/color/oci-icon-color.png?raw=true" width=30x>|[OCI Image][image-spec]|`application/vnd.oci.image.config.v1+json`|
+|<img src="https://github.com/helm/helm-www/blob/master/themes/helm/static/img/apple-touch-icon.png?raw=true" width=30x>|[Helm Chart](https://helm.sh)|`application/vnd.cncf.helm.chart.config.v1+json`|
+|<img src="https://github.com/sylabs/singularity/blob/master/docs/logos/singularity_v3.png?raw=true" width=30x>|[Singularity][singularity], by [Sylabs][sylabs]|`application/vnd.sylabs.sif.config.v1+json`|
+
+## Authoring an OCI Artifacts
+
+Authoring Artifacts involves the following steps:
+
+1. [Define a unique Artifact type](#defining-a-unique-artifact-type)
+1. Define the format for other tools to operate upon the type.
+1. Define human elements, such as an icon and localized string to be displayed to users.
+1. Optional: Register the unique `mediaType` with [IANA][iana-register-media-types].
+1. Optional: Publish as a [well-known type][def-well-known-types] for registries to consume.
+
+### Defining a Unique Artifact Type
+
+A unique artifact is similar to defining a file extension. Defining a unique Artifact allows various tools to know how to uniquely work with the type. It allows a registry to display the type, and vulnerability scanners a means to know if and how they should scan the contents.
+
+Defining a unique type involves uniqueness for computer processing, and uniqueness for humans. See [Defining Artifact Publisher Manifests](#optional-defining-artifact-publisher-manifests) for human readability.
+
+For computer processing, artifacts are defined by setting the `manifest.config.mediaType` to a globally unique value.
+
+> **Note:** The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be run and instanced by [docker][docker], [containerd][containerd] and other [OCI Image][image-spec] runtimes and tool chains.
+
+Each new artifact type MUST be uniquely defined. See [registering unique types with IANA](#registering-unique-types) for registering unique mediaTypes.
```

Comment:
A few lines above, we state:

```Optional: Register the unique `mediaType` with [IANA][iana-register-media-types].```

If it needs to be uniquely defined, how are we going to enforce this without registration?

Reply:
In the **Optional: Publish as a well-known type for registries to consume.** I call out well-known types MUST be registered with IANA. If a registry operator has a type unique to them, and they don't expect it to be used on other registries, they may choose to register it, or not.
For instance, we're moving ACR Tasks logs to be an artifact type, to benefit from double compression, geo-replication and auto-purge. We don't expect this type to be used on other registries. We may register it, but we won't publish it as a well known type. It's more a matter of grabbing a name if someone really needs it. 
I don't expect anyone to grab, or try to register `application/vnd.microsoft.azure.task.logs.config.v1`

Reply:
@SteveLasker All well and good until someone starts mirroring from Azure to GCR and then now its used everywhere ;)